src/mixins.coffee

```diff
@@ -64,3 +64,75 @@ Ember.Widgets.BodyEventListener = Ember.Mixin.create
   _removeDocumentHandlers: ->
     $(@get('bodyElementSelector')).off "click", @_clickHandler
     @_clickHandler = null
+
+Ember.Widgets.TabbableModal = Ember.Mixin.create
+  currentFocus: null
+
+  _focusTabbable: ->
+     # Set focus to the first match:
+     # 1. The saved focused element
+     # 2. First element inside the dialog matching [autofocus]
+     # 3. Tabbable element inside the content element
+     # 4. The close button (has class "close")
+    hasFocus = [@get 'currentFocus']
+    if hasFocus[0] == null or not $.contains(@$()[0], hasFocus[0])
```

Comment:
Is it from style guide or is there any other reason making is better than ==? I was also curious about that but hasn't got good answer.